functions.sh.in

```diff
@@ -92,8 +96,10 @@ _print_repository_path() {
 }
 
 repo_to_repositories() {
-	local func="$1"
-	local path="$2"
+	local func path
+
+	func="$1"
+	path="$2"
```

Comment:
What is the purpose of splitting local declaration from assignments?

Reply:
It allows me to use a simple do nothing function to implement local in shells that do not support that keyword without losing the assignment.